cartridges/int_adyen_SFRA/cartridge/templates/default/redirectHPP.isml

```diff
@@ -30,10 +30,22 @@
     var inputType: String = Debug ? 'text' : 'hidden';
 
     if (directoryLookup) {
-        var url: String = "https://test.adyen.com/hpp/skipDetails.shtml";
-        if (Mode == "LIVE") {
-            url = "https://live.adyen.com/hpp/skipDetails.shtml";
+
+        if (pdict.ParamsMap.brandCode == "klarna" || pdict.ParamsMap.brandCode == "klarna_b2b" || pdict.ParamsMap.brandCode == "afterpay_default"
```

Comment:
It might be nicer if you create 2 methods for this (e.g. isOpenInvoice() and isMolpay() methods). 
I already created the isOpenInvoice in the feature/checkout-api branch here, so you can reuse that one: 
https://github.com/Adyen/adyen-salesforce-commerce-cloud/blob/feature/checkout-api/cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds#L182

Reply:
Sounds great, that's what I was thinking as well - how can I reference the AdyenHelper in redirectHPP?